### PR TITLE
add link to training request form

### LINKED ISSUE #1116: Guidance: Add link to training request form
http://bit.ly/OCP-Training as used on this page: https://www.open-contracting.org/data-standard/documentation

### PR DESCRIPTION
**Pull request by Ravf95**
Closes #1116

### CODE REVIEW

File: docs/guidance/index.md [hunk 1]
```diff
@@ -17,4 +17,4 @@ build
 publish
 ```
 
-Do you have questions or need assistance? Contact the [OCDS Helpesk](../support/index).
+Do you have questions or need assistance? Contact the [OCDS Helpdesk](../support/index) or you can fill out the [Training Request Form](http://bit.ly/OCP-Training).
```
Comment (jpmckinney) [inline docs/guidance/index.md]: Thanks for correcting the typo. I think the link works better on this page: https://standard.open-contracting.org/latest/en/guidance/design/

Changing:

```md
Action: [Contact the OCDS Helpdesk](https://standard.open-contracting.org/latest/en/support/) to arrange an online introductory training for your technical team, in English or Spanish.
```

to:

```md
Action: [Contact the OCDS Helpdesk](https://standard.open-contracting.org/latest/en/support/) or [complete this form](http://bit.ly/OCP-Training) to arrange an online introductory training for your technical team, in English or Spanish.
```

Comment (Ravf95) [reply docs/guidance/index.md]: I agree, I'll apply the suggestion, thanks!

File: docs/guidance/design.md [hunk 2]
```diff
@@ -106,7 +106,7 @@ To achieve success, an OCDS implementation team ought to include the following r
 
 **Action:** Ask everyone in your team to [read the Primer](../../primer/index).
 
-**Action:** [Contact the OCDS Helpdesk](../../support/index) to arrange an online introductory training for your technical team, in English or Spanish.
+**Action**: [Contact the OCDS Helpdesk](https://standard.open-contracting.org/latest/en/support/) or [complete this form](http://bit.ly/OCP-Training) to arrange an online introductory training for your technical team, in English or Spanish.
```
Comment (jpmckinney) [inline docs/guidance/design.md]: Please leave the first link as it was.

Comment (Ravf95) [reply docs/guidance/design.md]: oops, I fix it

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**